include/gbinder_types.h

```diff
@@ -144,6 +145,16 @@ typedef struct gbinder_hidl_memory {
 #define GBINDER_HIDL_MEMORY_NAME_OFFSET (24)
 G_STATIC_ASSERT(sizeof(GBinderHidlMemory) == 40);
 
+typedef enum gbinder_fmq_type {
+    GBINDER_FMQ_TYPE_SYNC_READ_WRITE = 1,
+    GBINDER_FMQ_TYPE_UNSYNC_WRITE
+} GBINDER_FMQ_TYPE; /* Since 1.1.14 */
+
+typedef enum gbinder_fmq_flags {
+    GBINDER_FMQ_FLAG_CONFIGURE_EVENT_FLAG = 0x1,
+    GBINDER_FMQ_FLAG_NO_RESET_POINTERS    = 0x2
+} GBINDER_FMQ_FLAGS; /* Since 1.1.14 */
+
```

Comment:
Could these enums be in `gbinder_fmq.h` or do they have to be here?

Reply:
Moved.